visual-inspection/inspection-frontend/src/components/InspectImage.vue

```diff
@@ -2,7 +2,10 @@
   <div class="inspector">
     <MultiBounce v-if="!prediction"
                  v-bind:numberOfDots="3"/>
-    <p v-show="prediction && currentPrediction">{{ $t('answer', { object: prediction }) }}</p>
+     <p v-if="show_probability_percentage"
+        v-show="prediction && currentPrediction">{{ $t('answer1', {percentage: probability}, {object: prediction}) }}</p>
```

Comment:
```suggestion
        v-show="prediction && currentPrediction">{{ $t('answer1', {percentage: probability, object: prediction}) }}</p>
```

So sollte das gehen, vgl. https://vue-i18n.intlify.dev/guide/essentials/syntax.html#literal-interpolation